sdk/machinelearningservices/Azure.ResourceManager.MachineLearning/src/autorest.md

```diff
@@ -0,0 +1,93 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+
+azure-arm: true
+library-name: MachineLearning
+require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/0bff4d0f259847b1fc97a4ca8f98b8c40d672ba5/specification/machinelearningservices/resource-manager/readme.md
+tag: package-2022-02-01-preview
+modelerfour:
+    lenient-model-deduplication: true
+clear-output-folder: true
+namespace: Azure.ResourceManager.MachineLearning
+skip-csproj: true
+no-property-type-replacement: 
+- ResourceId
+- VirtualMachineImage
+directive:
```

Comment:
Please add the `rename rule` part here, I saw some acronyms that don't meet our naming convention.